csp-mixin/panels/azurequeuestore.libsonnet

```diff
@@ -0,0 +1,174 @@
+local g = import '../g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+{
+  new(this): {
+    // Azure Queue Storage
+    _tableCommon(valueName)::
+      g.panel.table.queryOptions.withTransformations([
+        g.panel.table.queryOptions.transformation.withId('filterFieldsByName')
+        + g.panel.table.queryOptions.transformation.withOptions({
+          include: {
+            pattern: 'resourceName.*|Value.*',
+          },
+        }),
+        g.panel.table.queryOptions.transformation.withId('joinByField')
+        + g.panel.table.queryOptions.transformation.withOptions({
+          byField: 'resourceName',
+          mode: 'outer',
+        }),
+      ])
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'resourceName',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Storage account',
+            },
+            // TODO: The link "works" but does not refresh the dashboard upon following the link. Surely this is solvable?
+            // {
+            //   id: 'links',
+            //   value: [
+            //     {
+            //       title: 'Filter to bucket',
+            //       url: '/d/%(uid)s/%(uid)s?${datasource:queryparam}&var-%(bucketLabel)s=${__data.fields.%(bucketLabel)s}' % {
+            //         uid: this.config.uid + '-blobstorage',
+            //         bucketLabel: this.config.blobStorage.bucketLabel,
+            //       },
+            //     },
+            //   ],
+            // },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: valueName,
+            },
+          ],
+        },
+      ]),
+
+    availabilityTs:
+      this.signals.azurequeuestore.availability.asTimeSeries()
+      + g.panel.timeSeries.options.legend.withDisplayMode('table')
+      + g.panel.timeSeries.options.legend.withPlacement('bottom'),
+    availabilityStat:
+      this.signals.azurequeuestore.availability.asStat(),
+    queueCount:
+      this.signals.azurequeuestore.queueCount.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('purple'),
+    messageCountTotal:
+      this.signals.azurequeuestore.messageCountTotal.asStat()
+      + g.panel.stat.standardOptions.color.withMode('fixed')
+      + g.panel.stat.standardOptions.color.withFixedColor('purple'),
```

Comment:
We are showing counts in gray. Can you do the same?

<img width="1657" alt="Screenshot 2024-09-26 at 17 26 07" src="https://github.com/user-attachments/assets/73b83055-4460-4c1b-bc0b-09ca258b995e">

<img width="829" alt="Screenshot 2024-09-26 at 17 26 37" src="https://github.com/user-attachments/assets/8d15c712-3c8c-46fa-a2e3-5024f4e1b249">

Reply:
Awww man.. Purple is so pretty tho ;)

I set it to `text` to be consistent.